tools/WinObjC.Packaging/WinObjC.Packager.targets

```diff
@@ -71,10 +71,10 @@
       <PackageFile Include="$(MSBuildThisFileDirectory)Package.Default.targets" Condition="Exists('$(MSBuildThisFileDirectory)Package.Default.targets')">
         <PackagePath>build\$(PackageId).Default.targets</PackagePath>
       </PackageFile>
-      <PackageFile Include="$(MSBuildProjectDirectory)\$(PackageId).Custom.targets" Condition="Exists('$(MSBuildProjectDirectory)\$(PackageId).Custom.targets')">
```

Comment:
PackageId == winobjc.facebooksdk
MSBuildProjectName == facebooksdkpackage
So we were using the wrong file name when trying to package the facebooksdkpackage.custom.targets/props.